app/src/main/java/org/stacks/app/data/interfaces/IIdentityRepository.kt

```diff
@@ -0,0 +1,25 @@
+package org.stacks.app.data.interfaces
+
+import dagger.Binds
+import dagger.Module
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ApplicationComponent
+import kotlinx.coroutines.flow.Flow
+import org.stacks.app.data.IdentityModel
+import org.stacks.app.data.IdentityRepository
+
+interface IIdentityRepository {
```

Comment:
Since most places will be using the Interface instead of the implementation, I usually prefer the interface to have the "clean" name `IdentityRepository` and the implementation to have the ugly one: `SharedPreferencesIdentityRepository`.